### ISSUE TITLE
Multiple primitives tutorial (or basic anatomy of an OpenGL renderer)

### ISSUE THREAD

**Issue by gnzlbg** [Overv/Open.GL#32]
I just finished your tutorials and they are great. I noticed, however, that most of the examples deal with a single primitive. It would be awesome to have a tutorial explaining how to set up the pipeline for working with multiple different primitives. I guess that drawing 2 cubes and 2 tetras (one of each moving) should be enough to explain:
- how to set up multiple VAOs, with different EBOs, in an initialization step,
- how to update some of the model matrices in a loop,
- how to draw the different VAOs in a loop.

I think such a tutorial could be a good complement to the rest. It would give a feeling about how a "real-world" OpenGL program looks like.

Bests,
Gonzalo

**Comment by blackwolf12333** [comment]
Yes, I really need this too. I have been trying to figure this out by myself, but it's hard...
